src/uct/ib/rc/base/rc_iface.c

```diff
@@ -380,14 +380,14 @@ ucs_status_t uct_rc_iface_fc_handler(uct_rc_iface_t *iface, unsigned qp_num,
         }
     }
 
-    if (fc_hdr & UCT_RC_EP_FC_FLAG_SOFT_REQ) {
+    if (ep && (fc_hdr & UCT_RC_EP_FC_FLAG_SOFT_REQ)) {
```

Comment:
why it could be NULL? and shouldn't we exit sooner in this case? e.g see line 361